### ISSUE TITLE
Talkback accessibility

### ISSUE THREAD

**Issue by davidbtc2009** [oxen-io/session-android#850]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

Talkback screen reader reads the profile button and search button on the main screen as "unlabeled" and the "+" icon cannot even be focused. It goes from the last chat in the list back to the unlabeled profile button. Once the "+" icon is clicked visually the globe icon, message bubble icon, snd group icons are also not able to be focused in order to be used by talkback. When inside a chat the "+" button and microphone are not usable by talkback as they cannot be focused.

**To reproduce**

1. Open Settings > Accessibility > Talkback and turn it on
2. Open session
3. Swipe through the icons on the main screen
4. Then open a chat and swipe through the chat screen

**Smartphone (please complete the following information):**

 - Device: Motorola One 5G Ace
 - OS: Android 11
 - Version of Session or latest commit hash: 1.11.17 from F-Droid Session repo

**Comment by KeeJef** [comment]
Might we worth tracking this @burtonemily

**Comment by KeeJef** [comment]
Some of these screens have changed since this issue was filed, but could be a good first issue to correct any discrepancies here

**Comment by shortwavesurfer2009** [comment]
@KeeJef updating this issue. The profile and search buttons on the chat list screen are still unlabeled. The "+" icon now focuses correctly but still reads as "unlabeled". When inside a chat the "+" icon to the left of the message box and the microphone )/send button (depending on if a message has been typed also cannot be focused.

**Comment by burtonemily** [comment]
> Might we worth tracking this @burtonemily

This should be fixed up when the Accessibilty ID's branch is merged @hjubb

**Comment by pbsinnett** [comment]
Hello,

I'm running Session 1.16.7 and experiencing the following accessibility bugs:

- Certain items are spoken as the place holders they represent.
1. "message body" when focusing on text messages
2. "username" when focused on it at the top.
3. Conversations are exhibiting this behavior from the conversation list.
4. In settings, "user settings" is spoken when focused on the item tapped for adding a profile photo.
- Unlabeled buttons.
1. When recording a voice message and sliding up to lock recording, the send button at bottom right is unlabeled.
2. In settings, the button at top right to show your QR code is unlabeled.
3. In-call screen buttons are unlabeled.

If I come across anything else, I'll comment here.

Thanks,
Blake

**Comment by hjubb** [comment]
Thanks @pbsinnett, is there any particular accessibility practices you would expect for those items? The actual content instead? We can probably mark those things down as something to improve on and bring it more in line with what standard practices would be

**Comment by shortwavesurfer2009** [comment]
The things that @pbsinnett mentioned are still issues as of now v1.17.4
